app/models/application_record.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+  self.implicit_order_column = 'created_at'
```

Comment:
I've never seen this pattern before and it's interesting. I didn't know about abstract classes in Ruby at all, and I'd never seen this to set a global sensible sort. I dig it.